src/app/components/ActiveUserLoanContainer/components/ExpandedRowMobile/index.tsx

```diff
@@ -15,12 +15,12 @@ export function ExpandedRowMobile(props) {
 
   return (
     <div style={{ opacity: '1' }} onClick={props.handleClick}>
-      <div className="tw-grid tw-gap-8 tw--mx-4 tw-grid-cols-12 table-header sub-header">
```

Comment:
I think `tw--mx-4` means `-mx-4` (negative 1rem), was it decided to change to positive margin?

Reply:
Fixed it.